.github/workflows/pr-create-release-build.yml

```diff
@@ -122,92 +121,92 @@ jobs:
         if: ${{ needs.wait-checks.result == 'success' }}
         run: |
           gh pr comment ${{ needs.create-pr.outputs.pr_number }} --body "/lgtm"
-          gh pr edit ${{ needs.create-pr.outputs.pr_number }} --add-label lgtm
         env:
-         GH_TOKEN: ${{ github.token }}
+          GH_TOKEN: ${{ github.token }}
 
   wait-lgtm:
     runs-on: ubuntu-latest
-    needs: [ check-prev-tag,create-pr,wait-checks,comment-lgtm ]
+    needs: [check-prev-tag, create-pr, wait-checks, comment-lgtm]
     outputs:
       has_lgtm: ${{ steps.wait-lgtm-label.outputs.has_lgtm }} 
+      has_lgtm: ${{ steps.wait-lgtm-label.outputs.has_lgtm }} 
 
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
+      has_lgtm: ${{ steps.wait-lgtm-label.outputs.has_lgtm }}
 
-      - name: Wait for lgtm label
+    steps:
+      - name: Wait for LGTM label
+        uses: nick-fields/retry@v3
         id: wait-lgtm-label
-        run: |
-          for i in {1..60}; do
+        with:
+          timeout_minutes: 120
+          max_attempts: 60

```

Comment:
I guess GHA actions can be triggered based on events, this will be safer than a retry loop, wdyt?

Reply:
Yes we dont need wait-lgtm i had added this bit of code while testing on the Local will remove and push the code